chore: Add article about retrieving the list of default images for Im… (#2795)
codenvy/che-docs: modules/administration-guide/pages/installing-kubernetes-image-puller.adoc

:_content-type: PROCEDURE
:description: Installing {image-puller-name}
:keywords: overview, installing, image-puller
:navtitle: Installing {image-puller-name}

[id="installing-{image-puller-name}"]
= Installing {image-puller-name}

> This section contains instructions for installing the {image-puller-name} for different use cases.

Reviewer: ```suggestion
Follow the instructions below to install the {image-puller-name} for different use cases.
```
